docs/source/logging/logging.md

```diff
@@ -1,54 +1,125 @@
 
-The project-side logging configuration also ensures that [logs emitted from your project's logger](#perform-logging-in-your-project) should be shown if they are `INFO` level or above (as opposed to the Python default of `WARNING`).
+```python
+log.error("[bold red blink]Important error message![/]", extra={"markup": True})
+```
 
-We now give some common examples of how you might like to change your project's logging configuration.
+## How to customise Kedro logging
 
 ### Using `KEDRO_LOGGING_CONFIG` environment variable
-
-`KEDRO_LOGGING_CONFIG` is an optional environment variable that you can use to specify the path of your logging configuration file, overriding the default Kedro's `default_logging.yml`.
-
-To use this environment variable, set it to the path of your desired logging configuration file before running any Kedro commands. For example, if you have a logging configuration file located at `/path/to/logging.yml`, you can set `KEDRO_LOGGING_CONFIG` as follows:
+To customise logging, specify the path of your logging configuration file by setting the environment variable `KEDRO_LOGGING_CONFIG` to override the default, which is Kedro's `default_logging.yml`. We recommend you put `logging.yml` inside the `conf` folder. For example, you can set `KEDRO_LOGGING_CONFIG` as follows:
 
 ```bash
-export KEDRO_LOGGING_CONFIG=/path/to/logging.yml
+export KEDRO_LOGGING_CONFIG=<project_root>/conf/logging.yml
 ```
 
 After setting the environment variable, any subsequent Kedro commands will use the logging configuration file at the specified path.
 
 ```{note}
 If the `KEDRO_LOGGING_CONFIG` environment variable is not set, Kedro will default to using the logging configuration file at the project's default location of  Kedro's `default_logging.yml`.
 ```
-### Disable file-based logging
 
-You might sometimes need to disable file-based logging, e.g. if you are running Kedro on a read-only file system such as [Databricks Repos](https://docs.databricks.com/repos/index.html). The simplest way to do this is to delete your `conf/base/logging.yml` file. With no project-side logging configuration specified, Kedro uses the default framework-side logging configuration, which does not include any file-based handlers.
+### Show DEBUG level messages
+To see `DEBUG` level messages, change the level of logging in `logging.yml`:
 
-Alternatively, if you would like to keep other configuration in `conf/base/logging.yml` and just disable file-based logging, then you can remove the file-based handlers from the `root` logger as follows:
-```diff
- root:
--  handlers: [console, info_file_handler]
-+  handlers: [console]
+```yaml
+loggers:
+  kedro:
+    level: INFO
+
+  your_python_pacakge:
+    level: DEBUG  # Change this to DEBUG
+```
+
+By changing the level value to `DEBUG` for the desired logger (e.g., <your_python_package>), you will start seeing `DEBUG` level messages in the log output.
```

Comment:
```suggestion
By changing the level value to `DEBUG` for the desired logger (e.g., `<your_python_package>`), you will start seeing `DEBUG` level messages in the log output.
```

Reply:
The example doesn't make much sense here because if you just copy and paste this code into a new logging.yml file it won't work (you'd need to set the `version` key as a bare minimum also or Python won't load the config).

I'd put it after the bit where you show the whole "advanced" template in the next section and then do as a `diff` so that it's obvious what lines you're changing.

Reply:
Does this work now I've adjusted the sectioning so it falls as a subsection of the "How to customise" and is _after_ the environment variable is set to point to `logging.yml` ? I didn't understand your comment about setting the version key.

Reply:
@stichbury I think the confusion here is the YAML example is only showing part of the file, the other part of the documentation is showing the `diff` instead.

A minimal `logging.yml` file need to have a `version` key which is required by the Python logging configuration. I will do as Antony suggested to add `+++` `---` to indicate it is a change that need to be applied on the file instead of just replacing the file.